tsconfigdoc.json

```diff
@@ -296,6 +300,12 @@
     {
       "path": "./packages/ui-components-extension"
     },
+    {
+      "path": "./packages/user"
+    },
+    {
+      "path": "./packages/user-extension"
+    },
```

Comment:
Noisy changes back again - I advice you to remove the two folders locally to avoid this.

```suggestion
```

Reply:
Thanks!